src/fragments/lib/graphqlapi/js/working-with-files.mdx

```diff
@@ -0,0 +1,910 @@
+The Storage and GraphQL API categories can be used together to associate a file, such as an image or video, with a particular record. For example, you might create a `User` model with a profile picture, or a `Post` model with an associated image. With Amplify's GraphQL API and Storage categories, you can reference the file within the model itself to create an association.
+
+To get started, run the following in an existing Amplify project:
+
+```bash
+npm i @aws-amplify/ui-react  #For Authenticator component
+amplify add auth             #Select default configuration
+amplify add storage          #Select S3 with read/write access
+amplify add api              #Select Cognito User Pool for authorization type
+```
+
+When prompted, use the following schema:
```

Comment:
I think we should be a bit more specific that you should find your schema .graphql file, and where it is at - and update it.

Reply:
@abdallahshaban557 - I'm not absolutely opposed, but I was attempting to not re-explain items that are found elsewhere in the docs (such as in the `Getting Started` guides). Given that this isn't a separate tutorial, and is found in the context of our docs, I'm inclined to say no. However, it's a minor update, so I'm okay either way.

Reply:
I would rather we mention it here IMO. It would make this section a lot clearer.

Reply:
I should also add that the CLI prompts you to open the file for editing, so this may be redundant.

Reply:
Added this in, marked as resolved